Description: Disable new mocha
 needs mocha >= 3
--- a/test/validation.js
+++ b/test/validation.js
@@ -591,7 +591,7 @@
         })
         .parse()
     })
-
+/*
     // addresses: https://github.com/yargs/yargs/issues/849
     it('succeeds when demandOption is true and valid choice is provided', (done) => {
       yargs('one -a 10 marsupial')
@@ -613,7 +613,7 @@
         })
         .parse()
     })
-
+*/
     // addresses: https://github.com/yargs/yargs/issues/849
     it('fails when demandOption is true and choice is not provided', (done) => {
       yargs('one -a 10 marsupial')
