Description: drop cross-spawn
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-01-13

--- a/test/integration.cjs
+++ b/test/integration.cjs
@@ -1,7 +1,7 @@
 'use strict';
 /* global describe, it, before, after */
 
-const spawn = require('cross-spawn');
+const spawn = require('child_process').spawn;
 const path = require('path');
 const which = require('which');
 const rimraf = require('rimraf');
